traveler-swift-core/TravelerKit/Extensions/Date+Day.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Date+Day.swift
+//  TravelerKit
+//
+//  Created by Rakin Hoque on 2019-11-26.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    public func minTimeOfDay() -> Date {
```

Comment:
Where in the code are you using these functions and for which purpose?

Reply:
These are being used in the UI SDK (UI PR).  These are used to check if an ItineraryItem correctly fits within the Itinerary Date Range.

Reply:
Moved to UI SDK in UI PR